js/util/request_form/domify.jsx

```diff
@@ -0,0 +1,14 @@
+/*
+ * domify
+ *
+ * Converts text to react elements. **Only use for known-safe content.**
+ */
+import React from 'react';
+
+function domify(content) {
+  /* eslint-disable react/no-danger */
+  return <div dangerouslySetInnerHTML={{ __html: content }} />;
```

Comment:
the name of this cracks me up every time I come across it.

Reply:
Don't laugh. This is serious. Well, maybe it should be `seriouslySetInnerHTML` 😀